modules/olm-understanding-operator-catalog-images.adoc

```diff
@@ -39,10 +40,10 @@ The following catalogs are distributed by Red Hat:
 |`community-operators`
 |`registry.redhat.io/redhat/community-operator-index:latest`
 |Software maintained by relevant representatives in the link:https://github.com/operator-framework/community-operators[operator-framework/community-operators] GitHub repository. No official support.
-
 |===
+endif::[]
 
-As catalogs are updated, the latest versions of Operators change, and older versions may be removed or altered. This behavior can cause problems maintaining reproducible installs over time. In addition, when OLM runs on an {product-title} cluster in a restricted network environment, it is unable to access the catalogs from Quay.io directly to pull the latest content.
+As catalogs are updated, the latest versions of Operators change, and older versions may be removed or altered. In addition, when OLM runs on an {product-title} cluster in a restricted network environment, it can be unable to access the catalogs directly from the Internet to pull the latest content.
```

Comment:
s/can be/is ?

Reply:
I think I was leaving it vague, depending on their "restricted network", but for these purposes I'll just change it to "is".